catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -99,6 +99,8 @@
      */
     public static final AttributeType<Short> SHORT_TYPE;
 
+    public static final AttributeType<HashMap> HASHMAP_TYPE;
```

Comment:
I don't think you can just define a new type here anyways, since these attribute formats correspond to the supported types in Solr, or at least our dynamic schema.